core/primitives-core/src/types.rs

```diff
@@ -19,7 +19,7 @@ pub type BlockHeight = u64;
 /// Height of the epoch.
 pub type EpochHeight = u64;
 /// Shard index, from 0 to NUM_SHARDS - 1.
```

Comment:
Please, reference the NEP and I also suggest to clarify that `ord` is "ordinal number", just to so any maintainer can easily get the context here.